source/04-reports/04-item-reports.rst

```diff
@@ -7,7 +7,7 @@
 Item Reports
 ------------
 
-Item Reports provide a summary of activity related to content at the item level and provide a means of evaluating the impact an item has for an institution’s patrons.
+Item Reports provide a summary of activity related to content at the item level and provide a means of evaluating the impact an item has for an institution’s patrons. To better facilitate open access reporting, Release 5.1 encourages publishers with open access content to provide the Item Report. 
```

Comment:
So far we've always used the term Open Access (capitalized). I forgot to mention this in some other pul requests.

Reply:
Style question on open access vs Open Access - I'm seeing an increasing push towards dropping the capitalization as a signal of the normalization of OA within our community. Suggest resolving this within the copy-edit issue.